### PR TITLE
Fetch roi info

### PR DESCRIPTION
**Pull request by bnmajor**
Provide the API that we need in order to retrieve information about the data bounds for the current region of interest, as defined by the cropping planes.

### CODE REVIEW

File: src/createViewer.js [hunk 1]
```diff
@@ -1254,6 +1257,37 @@ const createViewer = async (
     return context.maxConcurrency
   }
 
+  publicAPI.getloadedScale = name => {
+    if (typeof name === 'undefined') {
+      name = context.images.selectedName
+    }
```
Comment (PaulHax) [inline src/createViewer.js:1263]:
```suggestion
    const imageName = name ?? context.images.selectedName
```

**Review by PaulHax — APPROVED** [review]
Bueno!

**Review by thewtex — COMMENTED** [review]
Most excellent :woman_singer: !

Typo noted inline

File: src/ImJoyPluginAPI.js [hunk 2]
```diff
@@ -358,6 +358,18 @@ class ImJoyPluginAPI {
   getCompareImages(name) {
     return this.viewer.getCompareImages(name)
   }
+
+  getloadedScale(scale) {
```
[the remark this answers is not in the document]
Comment (bnmajor) [reply src/ImJoyPluginAPI.js]: Oh my :upside_down_face: Fixed!

**Review by thewtex — APPROVED** [review]
:sparkler:

**Pull request merged**